packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -73,59 +59,6 @@ const WalletTransfer = observer(({ is_wallet_name_visible, setIsWalletNameVisibl
         );
     }, [active_wallet?.loginid, active_wallet_name, from_account, to_account?.loginid]);
 
-    const [message_list, setMessageList] = React.useState<TMessageItem[]>([]);
```

Comment:
**Note:**
- Removed `message_list` state as it will be handled later when covering the the input validation errors for the input fields.
- Removed `validateAmount` function as its functionality will be handled in hooks later on.

Reply:
a little bit confused regarding "Removed `message_list` state" because on line 40 we leave it as it is